web3/utils/module_testing/eth_module.py

```diff
@@ -229,7 +230,11 @@ def test_eth_sendTransaction(self, web3, unlocked_account):
             ),
         ]
     )
-    def test_eth_sendRawTransaction(self, web3, raw_transaction, expected_hash):
+    def test_eth_sendRawTransaction(self,
+                                    web3,
+                                    raw_transaction,
+                                    funded_account_for_raw_txn,
```

Comment:
I think it's cleaner to set the fixture to `autouse=True` than to add a variable that goes unused.

Reply:
@carver sorry for the late reply. Should I still modify this part? or are you going to handle this:)?
Thank you for the merge anyway!

Reply:
No worries, it was pretty minor. No need to submit a PR for it. :) @mhchia